src/main/java/org/generationcp/middleware/api/germplasmlist/GermplasmListService.java

```diff
@@ -26,7 +26,7 @@ GermplasmListGeneratorDTO create(GermplasmListGeneratorDTO request, int status,
 	 * Inserts a list of multiple {@code GermplasmListData} objects into the database.
 	 *
 	 * @param data - A list of {@code GermplasmListData} objects to be persisted to the database. {@code GermplasmListData}
-	 *                           objects must be valid.
+	 *             objects must be valid.
 	 * @return Returns the ids of the {@code GermplasmListData} records inserted in the database.
 	 */
 	List<GermplasmListData> addGermplasmListData(List<GermplasmListData> data);
```

Comment:
Possible cleanup: Check if we can remove `addGermplasmListData` from service (seems to be used internally in GermplasmServiceImpl class and in test only)